### DISCUSSION TITLE
评论区上传webp图片成功，后端输出404!

### DISCUSSION THREAD

**Discussion by xiamuguizhi** [ArtalkJS/Artalk#106] [Q&A] [answered]
404 未找到
nginx

`https://artalk.xiamuyourenzhang.cn/static/images/20220417-203308.718.webp`

jpg图片正常显示

![image](https://user-images.githubusercontent.com/19650297/163714600-721d395d-3f97-4f22-be7e-a87a8f489788.png)

**Comment by qwqcode** [comment]
这可能是宝塔 Nginx 反代配置问题，官网测试没有问题

**Reply by qwqcode** [reply, marked as answer]
可参考：https://www.bt.cn/bbs/thread-57278-1-1.html

**Reply by xiamuguizhi** [reply]
@qwqcode  感谢 感谢，确实是nginx 反代配置问题，默认 `location  ~* \.(gif|png|jpg|css|js|woff|woff2)$` 添加上webp 即可！
